src/DependencyInjection/Compiler/AddFragmentsSupportPass.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2016 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+namespace Contao\CoreBundle\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+/**
+ * Adds the composer packages and versions to the container.
+ *
+ * @author Yanick Witschi <https://github.com/toflar>
+ */
+class AddFragmentsSupportPass implements CompilerPassInterface
```

Comment:
this class name is rather strange for what it does. Wouldn't we be better of by just having a `disabled` method on the `AddToSearchIndexListener` which can be called by any controller? Or just the opposite which is only called in our frontend controller?